bigchaindb/web/views/transactions.py

```diff
@@ -48,12 +48,22 @@ def get(self):
         parser.add_argument('operation', type=parameters.valid_operation)
         parser.add_argument('asset_id', type=parameters.valid_txid,
                             required=True)
+        parser.add_argument('last_tx', type=parameters.valid_bool,
+                            required=False)
         args = parser.parse_args()
-
         with current_app.config['bigchain_pool']() as bigchain:
             txs = bigchain.get_transactions_filtered(**args)
-
-        return [tx.to_dict() for tx in txs]
+            if args['last_tx'] and args['last_tx'] is True:
+                lastTX = None
+
+                for x in txs:
```

Comment:
What is going on here? Are you just trying to get the last element of the iterable like that?